pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -1185,7 +1185,7 @@ def scaled_pos_xyz(self):
         xyz = self.get_scaled_positions(wrap=False)
         return xyz[:, 0], xyz[:, 1], xyz[:, 2]
 
-    def get_extended_positions(self, width, return_indices=False, norm_order=2):
+    def get_extended_positions(self, width, return_indices=False, norm_order=2, positions=None):
```

Comment:
This feels more and more like a general utility for our analyse functions.  Do we expect people to call this manually?  If not I'd move it away from `Atoms` and into a non-public helper function in the analyse module.  Tell-tale sign for me is that the only thing this method needs from its instance is `positions` and `cells` and one of them can even be passed over by an argument.

Reply:
Actually we need it the most in `get_neighbors`. I guess the most elegant solution would be to derive `Interstitials` from `Atoms`, which I also thought about. The biggest problem however is that we don't have a proper definition for vacancies. Maybe it's time to raise this issue because we've talked about it some years ago with @liamhuber @jan-janssen @sudarsan-surendralal etc.